db/schema.sql

```diff
@@ -108,6 +108,44 @@ CREATE TABLE `memberinfo` (
   KEY `member_id` (`member_id`)
 );
 
+CREATE TABLE `policies` (
+  `policy_id` varchar(100) NOT NULL default '',
+  `title` text NOT NULL,
+  `description` text NOT NULL,
+  `image` varchar(200) default '',
+  `image_attrib` varchar(200) default '',
+  `image_license` varchar(200) default '',
+  `image_license_url` text,
+  `image_source` text,
+  UNIQUE KEY `policy_id` (`policy_id`)
+);
+
+CREATE TABLE `policydivisions` (
+  `division_id` varchar(100) NOT NULL,
+  `policy_id` varchar(100) NOT NULL default '',
+  `house` varchar(100),
```

Comment:
Do we want a separate division (or divisions) table with the division only bits?